service/src/test/java/bio/terra/workspace/db/TestDao.java

```diff
@@ -0,0 +1,78 @@
+package bio.terra.workspace.db;
+
+import bio.terra.common.db.WriteTransaction;
+import bio.terra.workspace.service.resource.exception.DuplicateResourceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Component;
+
+/** DAO in the test to contain direct access to the database */
+@Component
+public class TestDao {
```

Comment:
Maybe `RawDaoTestFixture`? `TestDao` looks like the name of a test suite.

Reply:
Good idea. I like it.